pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -312,8 +317,6 @@ func (r *ReconcileGrafanaDashboard) manageSuccess(dashboard *grafanav1alpha1.Gra
 // Handle error case: update dashboard with error message and status
 func (r *ReconcileGrafanaDashboard) manageError(dashboard *grafanav1alpha1.GrafanaDashboard, issue error) {
 	r.recorder.Event(dashboard, "Warning", "ProcessingError", issue.Error())
-	dashboard.Status.Phase = grafanav1alpha1.PhaseFailing
-	dashboard.Status.Message = issue.Error()
 
 	err := r.client.Status().Update(r.context, dashboard)
```

Comment:
Does this dashboard status update still make sense, now that the CRD type doesn't have a status block? There's another one on L309 also.

Reply:
True, I think the `.Status.` part is no longer necessary